{
	"name": "CVE-2023-52793",
	"threat_severity": "Low",
	"public_date": "2024-05-21T00:00:00Z",
	"bugzilla": {
		"description": "kernel: samples/bpf: syscall_tp_user: Fix array out-of-bound access",
		"id": "2282769",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2282769"
	},
	"cvss3": {
		"cvss3_base_score": "4.4",
		"cvss3_scoring_vector": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:N/I:N/A:H",
		"status": "draft"
	},
	"cwe": "CWE-125",
	"details": [
		"No description is available for this CVE."
	],
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2023-52793\nhttps://nvd.nist.gov/vuln/detail/CVE-2023-52793\nhttps://lore.kernel.org/linux-cve-announce/2024052157-CVE-2023-52793-18ab@gregkh/T"
	],
	"package_state": [
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:6",
			"fix_state": "Not affected",
			"package_name": "kernel",
			"product_name": "Red Hat Enterprise Linux 6"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:7",
			"fix_state": "Not affected",
			"package_name": "kernel",
			"product_name": "Red Hat Enterprise Linux 7"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:7",
			"fix_state": "Not affected",
			"package_name": "kernel-rt",
			"product_name": "Red Hat Enterprise Linux 7"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:8",
			"fix_state": "Not affected",
			"package_name": "kernel",
			"product_name": "Red Hat Enterprise Linux 8"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:8",
			"fix_state": "Not affected",
			"package_name": "kernel-rt",
			"product_name": "Red Hat Enterprise Linux 8"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:9",
			"fix_state": "Not affected",
			"package_name": "kernel",
			"product_name": "Red Hat Enterprise Linux 9"
		},
		{
			"cpe": "cpe:/o:redhat:enterprise_linux:9",
			"fix_state": "Not affected",
			"package_name": "kernel-rt",
			"product_name": "Red Hat Enterprise Linux 9"
		}
	],
	"upstream_fix": "kernel 6.5.13, kernel 6.6.3, kernel 6.7",
	"csaw": false
}
